code-of-conduct.md

```diff
@@ -0,0 +1,165 @@
+# Adafruit CircuitPython Community Code of Conduct
+
+## Our Pledge
+
+In the interest of fostering an open and welcoming environment, we as
+contributors and maintainers pledge to making participation in our project and
+our community a harassment-free experience for everyone, regardless of age, body
+size, disability, ethnicity, gender identity and expression, level or type of
+experience, education, socio-economic status, nationality, personal appearance,
+race, religion, or sexual identity and orientation.
+
+## Our Standards
+
+We are committed to providing a friendly, safe and welcoming environment for
+all.
+
+On the Adafruit Discord server, please avoid using nicknames or avatar images
+that might detract from a friendly, safe and welcoming environment. This
+includes but is not limited to use of overtly sexual, vulgar, or otherwise
```

Comment:
Recommend removing the word "Overtly" as, realistically, we don't want subtle stuff either. And it also removes the potential for people to justify things to themselves by claiming "It wasn't overt!"